src/vs/workbench/browser/contextkeys.ts

```diff
@@ -390,3 +386,48 @@ export class WorkbenchContextKeysHandler extends Disposable {
 		this.temporaryWorkspaceContext.set(isTemporaryWorkspace(this.contextService.getWorkspace()));
 	}
 }
+
+export class EditorPartContextKeysHandler extends Disposable {
```

Comment:
I suggest to move this into `editorPart.ts` where it is being instantitated or inline it. I see no big benefit of having it as a class tbh.

Reply:
This was to align it with `WorkbenchContextKeysHandler` and not clutter the editor part as it might grow rather large with time. But don't mind moving it into editorPart, makes some stuff easier potentially.